cas-server-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -110,7 +110,8 @@
         </servlet-class>
         <init-param>
             <param-name>contextConfigLocation</param-name>
-            <param-value>/WEB-INF/cas-servlet.xml, /WEB-INF/cas-servlet-*.xml</param-value>
+            <!-- Load the child application context. Start with the default, then modules, then overlays. -->
+            <param-value>/WEB-INF/cas-servlet.xml,classpath*:/META-INF/cas-servlet-*.xml,/WEB-INF/cas-servlet-*.xml</param-value>
```

Comment:
So here is our new rule for module configuration (and I'm thinking about the new OAuth / OpenID connect module I should build): I would just need to add a `cas-servlet-oauth-oidc.xml` if required, right?